### ISSUE TITLE
grpc-bridge example fails to compile go service

### ISSUE THREAD

**Issue by drobertduke** [lyft/envoy#327]
```
$> pwd
/tmp/lyft/envoy/examples/grpc-bridge
$> script/build
# github.com/lyft/envoy/examples/grpc-bridge/service/gen
../../../../gopath/src/github.com/lyft/envoy/examples/grpc-bridge/service/gen/kv.pb.go:89: undefined: grpc.SupportPackageIsVersion3
```
gRPC master has moved to SupportPackageIsVersion4 apparently https://github.com/grpc/grpc-go/blob/master/rpc_util.go#L519 
Another triumph for golang dependency management 😃

**Comment by mattklein123** [comment]
Related to #155. We need to automate building/testing the example containers.

**Comment by junr03** [comment]
fixed by #357
